net/src/Substrate.Gear.Client/Model/Types/Primitive/NonZeroU8.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Substrate.NetApi.Attributes;
+using Substrate.NetApi.Model.Types.Base;
+using Substrate.NetApi.Model.Types.Metadata.Base;
+using Substrate.NetApi.Model.Types.Primitive;
+
+namespace Substrate.Gear.Client.Model.Types.Primitive;
+
+/// <summary>
+/// NonZeroU8
+/// </summary>
+[SubstrateNodeType(TypeDefEnum.Composite)]
```

Comment:
Do we need this annotations?

Reply:
it present in the generated types
https://github.com/gear-tech/sails/blob/master/net/src/Substrate.Gear.Api/Api/Generated/Types/Base/NonZeroU32.cs

Reply:
Technically these types are not composites